### PR TITLE
Updated troubleshooting md

### PR DESCRIPTION
**Pull request by mprather**
## Change Description

Improved 'fprime-util: command not found' section. Related to #2837
Also added a link to the navbar to make it easier to find the FAQ.

## Rationale

Better documentation makes it easier to use the framework.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -23,11 +23,26 @@ Some of the F´ Python packages are built in a way that it is recommended to ins
...
-If installing without a virtual environment, PIP occasionally uses `$HOME/.local/bin` as a place to install user tools. Users running without virtual environments should add this directory to the path.
+There are at least three common reasons for this error.
+
+1. If the user is using a virtual environment and receives the ‘command not found’, the problem is likely caused by the environment not being sourced in a new terminal. Make sure to source the environment before running:
+    ```sh
+    # In MyProject
+    . fprime-venv/bin/activate
+    ```
+
+1. If the project folder was moved after the virtual environment was provisioned (e.g. project was originally setup in `/folderA/MyProject` and moved to `/someOtherFolder/MyProject`), various files in the original `fprime-venv` folder will contain references to the original path. The easist resolution is to remove and reprovision the virtual environment with the following:
```
Comment (thomas-bc) [inline docs/troubleshooting.md]: typo: `easist` -> `easiest`

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -23,11 +23,26 @@ Some of the F´ Python packages are built in a way that it is recommended to ins
...
+    # In MyProject
+    rm -rf fprime-venv
+    python -m venv fprime-venv
+    . fprime-venv/bin/activate
+    pip install -r fprime/requirements.txt
+    ```
+
+    > Some projects ship their own `requirements.txt`.  Install using that file if it exists.
+
+1. If installing without a virtual environment, PIP occasionally uses `$HOME/.local/bin` as a place to install user tools. Users running without virtual environments should add this directory to the path.
```
Comment (thomas-bc) [inline docs/troubleshooting.md:45]: Numbering is off - should be 1 - 2 - 3 and not 1 - 1 - 1

**Review by thomas-bc — CHANGES_REQUESTED** [review]
Typo and numbering issue - otherwise looks good to me!

File: docs/troubleshooting.md (hunk 2 above)
Comment (mprather) [reply docs/troubleshooting.md:45]: Did you know that list numbering is automatically handled by the md engine? If you use 1-1-1, then future reordering doesn't seem as messy in diff views.  See https://www.markdownguide.org/basic-syntax/#ordered-lists

I'll submit the fixes shortly.

Comment (thomas-bc) [reply docs/troubleshooting.md:45]: Oh wow I did not know haha. Today I learned! No need to fix then - thanks!

**Review by thomas-bc — APPROVED** [review]

**Review by LeStarch — APPROVED** [review]

**Pull request merged**